dashboard/app/models/script.rb

```diff
@@ -1125,55 +1125,63 @@ def fix_lesson_positions
   end
 
   def clone_migrated_unit(new_name, new_level_suffix: nil, destination_unit_group_name: nil, version_year: nil, family_name:  nil)
+    raise 'Script name has already been taken' if Script.find_by_name(new_name) || File.exist?(new_name)
```

Comment:
Here's the modified check to make sure we aren't deleting something that already exists. Does that look right?

Reply:
I think you want `File.exist?(Script.script_json_filepath(new_name)`.

Reply:
Ah, I see, it creates `dashboard/config/scripts_json/csp7-1999.script_json`. Thank you for checking that! Just tested the new version locally:
The script_json file exists, the Script doesn't exist in the database, and the error is thrown when trying to clone:
![image](https://user-images.githubusercontent.com/9142121/151582108-57f280e5-fe47-4651-a71b-eb993cffe516.png)